types/index.d.ts

```diff
@@ -1,10 +1,12 @@
-// TypeScript Version: 2.6
+// TypeScript Version: 2.8
 
 /**
  * Remove the variants of the second union of string literals from
  * the first.
  *
- * @see https://github.com/Microsoft/TypeScript/issues/12215#issuecomment-307871458
+ * TypeScript Documentation suggests that as of 2.8 you should use the built-in "Exclude" instead.
+ *
+ * @see https://github.com/Microsoft/TypeScript-Handbook/blame/master/pages/release%20notes/TypeScript%202.8.md#L245
```

Comment:
Let's either alias `Diff` to `Exclude` or just take it out; I think I'd be fine with the latter and bumping the major version since we're already making the breaking change of requiring TypeScript 2.8.

Reply:
Currently making it an Alias, but happy to delete it instead, please advise!

Reply:
Let's delete.

Reply:
Deleted!